open-machine-learning-jupyter-book/prerequisites/python-programming-advanced.md

```diff
@@ -1302,15 +1185,13 @@ class Person:
         return self.name
 ```
 
-The syntax for a derived class definition looks like this.
-
-The Base Class (in our case Person) must be defined in a scope containing the derived class definition. In place of a base class name, other arbitrary expressions are also allowed.
+The Base Class (in our case `Person`) must be defined in a scope containing the derived class definition. In place of a base class name, other arbitrary expressions are also allowed.

```

Comment:
Base is a class name or not? If it is, should be `Base`

Reply:
not a name